backend/open_webui/routers/system_prompts.py

```
@@ -0,0 +1,201 @@
+                response = await client.delete(url, headers=headers, timeout=10.0)
+            else:
+                raise HTTPException(
+                    status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
+                    detail=f"Method {method} not supported",
+                )
+
+            if not response.is_success:
+                try:
+                    error_data = response.json()
+                    detail = error_data.get("detail", f"HTTP {response.status_code}")
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)